ffmpeg/ffmpeg.go

```diff
@@ -107,41 +108,163 @@ type TranscodeResults struct {
 	Encoded []MediaInfo
 }
 
-func GetCodecInfo(fname string) (bool, string, string, error) {
+type PixelFormat struct {
+	RawValue int
+}
+
+const (
+	PixelFormatNone        int = C.AV_PIX_FMT_NONE
+	PixelFormatYUV420P     int = C.AV_PIX_FMT_YUV420P
+	PixelFormatYUYV422     int = C.AV_PIX_FMT_YUYV422
+	PixelFormatYUV422P     int = C.AV_PIX_FMT_YUV422P
+	PixelFormatYUV444P     int = C.AV_PIX_FMT_YUV444P
+	PixelFormatUYVY422     int = C.AV_PIX_FMT_UYVY422
+	PixelFormatNV12        int = C.AV_PIX_FMT_NV12
+	PixelFormatNV21        int = C.AV_PIX_FMT_NV21
+	PixelFormatYUV420P10BE int = C.AV_PIX_FMT_YUV420P10BE
+	PixelFormatYUV420P10LE int = C.AV_PIX_FMT_YUV420P10LE
+	PixelFormatYUV422P10BE int = C.AV_PIX_FMT_YUV422P10BE
+	PixelFormatYUV422P10LE int = C.AV_PIX_FMT_YUV422P10LE
+	PixelFormatYUV444P10BE int = C.AV_PIX_FMT_YUV444P10BE
+	PixelFormatYUV444P10LE int = C.AV_PIX_FMT_YUV444P10LE
+	PixelFormatYUV420P16LE int = C.AV_PIX_FMT_YUV420P16LE
+	PixelFormatYUV420P16BE int = C.AV_PIX_FMT_YUV420P16BE
+	PixelFormatYUV422P16LE int = C.AV_PIX_FMT_YUV422P16LE
+	PixelFormatYUV422P16BE int = C.AV_PIX_FMT_YUV422P16BE
+	PixelFormatYUV444P16LE int = C.AV_PIX_FMT_YUV444P16LE
+	PixelFormatYUV444P16BE int = C.AV_PIX_FMT_YUV444P16BE
+	PixelFormatYUV420P12BE int = C.AV_PIX_FMT_YUV420P12BE
+	PixelFormatYUV420P12LE int = C.AV_PIX_FMT_YUV420P12LE
+	PixelFormatYUV422P12BE int = C.AV_PIX_FMT_YUV422P12BE
+	PixelFormatYUV422P12LE int = C.AV_PIX_FMT_YUV422P12LE
+	PixelFormatYUV444P12BE int = C.AV_PIX_FMT_YUV444P12BE
+	PixelFormatYUV444P12LE int = C.AV_PIX_FMT_YUV444P12LE
+)
+
+type ColorDepthBits int
+
+const (
+	ColorDepth8Bit  ColorDepthBits = 0
```

Comment:
Maybe add a comment detailing how these values are related to actual number of bits?